views/index.pug

```diff
@@ -0,0 +1,84 @@
+html
```

Comment:
I don't see routes for designers, is the only way to create a designer to create a bag for that designer? I would expect that bags and designers are two separate models. If they aren't then you have single dimension data and you'll need to add to it to meet the multi-dimensional requirement for part 2.